src/main/java/twilight/MarkingCommand.java

```diff
@@ -0,0 +1,32 @@
+package twilight;
+
+import java.io.IOException;
+
+public class MarkingCommand extends Command {
+    protected int type;
```

Comment:
Since type only has the value of 1 and 2, will it be better to declare an enum for it?

Reply:
The only problem with this, is that I would have to define the enum in the task class and then check the type of input to convert to an enum and it feels like the enum adds extra steps instead of solving anything, but this is probably because I am unfamiliar with enums and can't see the benefit.